app.js

```diff
@@ -1,12 +1,25 @@
 const express = require("express");
-const { getCategories, getReviews } = require("./controllers/controller.js");
+const {
+  getCategories,
+  getReviews,
+  getReviewById,
+} = require("./controllers/controller.js");
 
 const app = express();
 
 app.get("/api/categories", getCategories);
 
 app.get("/api/reviews", getReviews);
 
+app.get("/api/reviews/:review_id", getReviewById);
+
+// for 400's
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+});
+
 // for 500's
 app.use((err, req, res, next) => {
   console.log(err);
```

Comment:
if it's for a 500 we don't just want to send a console.log but also maybe send a body saying internal server error back!